src/modules/user/Register/index.jsx

```diff
@@ -0,0 +1,65 @@
+import { useState } from 'react';
+import { useUser} from '../UserContext';
+import { useNavigate } from 'react-router-dom';
+import "./styles.css";
+
+export const Register = () => {
+  const [userName, setUserName] = useState("");
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  const { login } = useUser();
+  const navigate = useNavigate();
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      // Burada API çağrısı yapılacak ve kayıt işlemi gerçekleştirilecek
+      const userData = { userName, email, token: "fake-token" }; //şifre saklanmıyor şifre token olarak görünüyor
```

Comment:
It would be better to use a unique token value here. Using Date.now() generates a timestamp (A timestamp is a sequence of characters or encoded information identifying when a certain event occurred, usually giving date and time of day, sometimes accurate to a small fraction of a second.) which makes the value unique. We don't use it in real life projects but for hobby projects it's a good enough solution:
```suggestion
      const userData = { userName, email, token: Date.now() }; //şifre saklanmıyor şifre token olarak görünüyor
```